assets/sass/components/global/_googlesitekit-header.scss

```diff
@@ -51,7 +66,68 @@
 		}
 	}
 
-	.googlesitekit-header__dropdown {
-		margin-right: -8px;
+	.googlesitekit-dropdown-menu {
+		display: inline-block;
+
+		.mdc-button__icon {
+			margin-right: 12px;
+		}
+
+		.mdc-button__icon,
+		.mdc-button__icon--image {
+			height: 28px;
+			width: 28px;
+		}
+
+		.mdc-button__label {
+			font-weight: $fw-primary-normal;
+		}
+	}
+
+	.googlesitekit-date-range-selector {
+		margin-right: 8px;
+
+		@media (max-width: $bp-xsmallOnly) {
+			margin-right: 0;
+		}
+
+		@media (min-width: $bp-wpAdminBarTablet) {
+			margin-right: 24px;
+		}
+
+		svg {
+			margin-right: 8px;
+		}
+
+		.mdc-button--dropdown {
+			padding-right: 25px;
```

Comment:
For values where we aren't referencing a variable, can you please annotate or clarify why we're using that value? We should always use existing variables where available and potentially add them where it makes sense over using an arbitrary value which is difficult to maintain.

Reply:
Switched to variables in almost all cases, added additional comments.